### PR TITLE
First cut of content.

### PR DESCRIPTION
**Pull request by mareuter**
To view the document locally do the following.

1. Clone this repo and switch to the branch
1. Create a new conda environment (I called mine technote) using Python3
1. Activate the previously create environment
1. cd to the clone if not already there
1. Run _pip install -r requirements.txt_
1. Run _make html_
1. Navigate your browser to generated documentation

I'm mainly looking for feedback on the flow of the document. However, I'm happy to discuss the finer points of what goes into the interface.

### CODE REVIEW

**Comment by cathypetry** [comment]
so far, if I go to

[cid:802CF7E5-5D35-4ACF-A1C2-88BB888D810E@lsst.local]

And I go to index.html - I just get - but there are no links.  Is this all?

[cid:EC0C3BBC-A733-4A3E-9AC9-ED2024020B14@lsst.local]


On Nov 11, 2016, at 9:54 AM, Michael Reuter <notifications@github.com<mailto:notifications@github.com>> wrote:


Assigned #1<https://github.com/lsst-sims/smtn-007/pull/1> to @cathypetry<https://github.com/cathypetry>.

File: index.rst [hunk 1]
```diff
@@ -1,51 +1,131 @@
...
-  those are automatically added.
+   **This technote is not yet published.**
 
-  Use the following syntax for sections:
+Introduction
+============
 
-  Sections
-  ========
+The Simulated Observatory Control System (SOCS) is designed to work with the LSST Scheduler. However, this does not mean that other Schedulers can't be developed and interfaced with SOCS in order to produce simulated surveys. This document details the required interfaces and specifications that will allow another Scheduler to be used with SOCS.
```
Comment (cathypetry) [inline index.rst]: Perhaps we could make this a more positive statement by replacing the second sentence with "...LSST Scheduler, and it was also designed with the intent that other Schedulers could be developed and interfaced with SOCS in order to produce simulated surveys."

File: index.rst [hunk 2]
```diff
@@ -1,51 +1,131 @@
...
+There are currently four topics that the LSST Scheduler sends to SOCS, all of which are required for SOCS to correctly function. It is not completely necessary to have all information within a topic filled. Each topic will now be discussed in detail.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Fields
+^^^^^^
 
-     Caption text.
+All information is currently mandatory due to the LSST Scheduler being responsible for the information. However, the new `Sims Survey Fields code`_ is destined end the need for a Scheduler to send this information as all code bases can benefit from the central resource. For mapping sky brightness and airmass, a field Id is required by SOCS. It does not currently offer HealPix support, however, one can work with the author to determine methodology for commensurate behavior.
```
Comment (cathypetry) [inline index.rst]: "... is destined end the..."  should be "...is destined to end the..." (I think)

File: index.rst [hunk 3]
```diff
@@ -1,51 +1,131 @@
...
-  _static/ directory. The reST syntax for adding the image is
+There are currently four topics that the LSST Scheduler sends to SOCS, all of which are required for SOCS to correctly function. It is not completely necessary to have all information within a topic filled. Each topic will now be discussed in detail.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Fields
+^^^^^^
 
-     Caption text.
```
Comment (cathypetry) [inline index.rst:32]: "field Id" should be "field ID" or "unique field identifier"

File: index.rst [hunk 4]
```diff
@@ -1,51 +1,131 @@
...
-  -----------
+SAL/DDS Communication
+---------------------
 
-  and
+This is by far the most stringent requirement for any Scheduler. The LSST Observatory Control System (OCS) communicates with all components via the Software Abstraction Layer (SAL)/Distributed Data Service (DDS) protocol. SOCS also communicates this way to interface with the LSST Scheduler. There is an Interface Control Document (ICD) which details currently defined the SAL/DDS topics for Scheduler communication. The `Scheduler ICD`_ is kept in LSST's DocuShare repository. If you do not have access to DocuShare, you can contact the author for a copy. The specification of the topics is kept with the `SAL code`_ on the ``feature/scheduler_xml_devel`` branch. Setup and building of the topic library can be found in the `SOCS documentation`_. **NOTE**: The SAL/DDS system requires running on a Linux environment. Mac support is in the works.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+From Scheduler Topics
```
Comment (cathypetry) [inline index.rst]: It took me a  second to interpret  what this meant - I like that it is short, however.   Maybe make it a hyphenated adjective:  "From-Scheduler Topics" ?  And the same for To Scheduler Topics.

File: index.rst [hunk 5]
```diff
@@ -1,51 +1,131 @@
...
+
+Observatory Model
+-----------------
+
+The current `Scheduler code`_ contains the only sanctioned observatory model for the project. It must be used to calculate all slew times for ranking targets. SOCS provides the configuration for the observatory via a DDS/SAL topic that any Scheduler must recieve, convert and pass that to the instantiation of the observatory model. Examples of use can be found in the `Scheduler code`_ repository. Again, all Schedulers **MUST** use this model even if no other code from the LSST Scheduler is used. It is possible that this code may be refactored into a separate repository for easier use.
+
+Sky Brightness Model
+--------------------
+
+SOCS and the Scheduler will be moving to the pre-calculated `Sky Brightness code`_ by the time version 1.0 of the combined system comes out. This is a project sanctioned model for sky brightness. This is the model that SOCS will use to determine the sky brightness at the time of observation. Any Scheduler should make use of this model to ensure commensurate sky brightness behavior between the two systems.
```
Comment (cathypetry) [inline index.rst]: If a non-LSST person is reading this, they may know know what "project sanctioned" means.  Maybe just change to "model sanctioned by the LSST Project."

File: index.rst [hunk 6]
```diff
@@ -1,51 +1,131 @@
...
+SAL/DDS Communication
+---------------------
+
+To Scheduler Topics
+~~~~~~~~~~~~~~~~~~~
+
+Scheduler Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Scheduler Driver and Scheduler configuration topics are used by the LSST Scheduler. However, other Scheduler's are not required to use them. If configuration of another Scheduler is required, please work with the SOCS author to ensure proper communication of parameters.
```
Comment (cathypetry) [inline index.rst]: "Scheduler's" should be "Schedulers"

File: index.rst [hunk 7]
```diff
@@ -1,51 +1,131 @@
...
+
+Clouds
+^^^^^^
+
+SOCS currently uses the OpSim3 cloud information. A Scheduler may have it's own cloud model and ignore the information coming from SOCS. However, the observation cloud information is taken from the OpSim3 values. If a Scheduler author wishes to have commensurate behavior, please work with the SOCS author to design a suitable mechanism for information gathering.
```
Comment (cathypetry) [inline index.rst]: "it's" should be "its"

**Comment by cathypetry** [comment]
Overall - great start!!!  It feels like you just jump into the details - which is okay if the prospective scheduler-writer already has some knowledge about how SOCS/Scheduler works already.  But if you envision that this will be a landing page or a launching pad page - then perhaps you could add a couple more lines after the first sentence and before the last sentence of the Introduction giving a larger context of what the SOCS/Scheduler does and the over all architecture.  You already have lots of references so I wouldn't replicate those, but maybe move the part about "The LSST Observatory Control System (OCS) communicates with all components via the Software Abstraction Layer (SAL)/Distributed Data Service (DDS) protocol. SOCS also communicates this way to interface with the LSST Scheduler." to the Introduction and follow up with generally what happens (or even show the diagram!) - e.g. information comes in, decisions are made, output goes out and stored in a db....  Just a few sentences - and could point to the SPIE paper too.

**Review by cathypetry — CHANGES_REQUESTED** [review]
All done!!

File: index.rst [hunk 8]
```diff
@@ -1,51 +1,131 @@
-..
-  Technote content.
+:tocdepth: 1
```
Comment (cathypetry) [inline index.rst:1]: Here is my test request for changes.

File: index.rst [hunk 9]
```diff
@@ -1,51 +1,131 @@
...
 
-  See https://developer.lsst.io/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+.. note::
```
Comment (cathypetry) [inline index.rst]: Heres' another recommendation.

**Review by cathypetry — CHANGES_REQUESTED** [review]
Here is my attempt at requesting your attention.

File: index.rst (hunk 8 above)
Comment (mareuter) [reply index.rst:1]: Change not necessary.

File: index.rst (hunk 9 above)
Comment (mareuter) [reply index.rst]: Change not necessary.

File: index.rst (hunk 2 above)
Comment (mareuter) [reply index.rst]: You are correct. Fixed.

File: index.rst (hunk 6 above)
Comment (mareuter) [reply index.rst]: You are correct. Fixed.

File: index.rst (hunk 7 above)
Comment (mareuter) [reply index.rst]: You are correct. Fixed.

File: index.rst (hunk 1 above)
Comment (mareuter) [reply index.rst]: I like what you said better as it is more positive. Changed.

File: index.rst (hunk 4 above)
Comment (mareuter) [reply index.rst]: Sounds like a good idea. Changed.

File: index.rst (hunk 5 above)
Comment (mareuter) [reply index.rst]: Sounds like a good clarification. Changed.

File: index.rst (hunk 3 above)
Comment (mareuter) [reply index.rst:32]: Comment looks a bit out of place, but this is fixed.

**Comment by mareuter** [comment]
For the final large comment, I moved the suggested line into the opening paragraph and included a citation to the SOCS SPIE paper.

**Review by cathypetry — APPROVED** [review]

**Pull request merged**